include/psen_scan/scanner.h

```diff
@@ -62,10 +65,12 @@ class Scanner : public vScanner
   MonitoringFrame previous_monitoring_frame_;   /**< Buffer for incoming Laserscanner data */
   std::unique_ptr<UDPInterface> udp_interface_; /**< Pointer to UDP Communication Interface */
 
-  MonitoringFrame fetchMonitoringFrame();
+private:
+  MonitoringFrame fetchMonitoringFrame(std::chrono::steady_clock::duration timeout = FETCH_FRAMES_TIMEOUT);
```

Comment:
Adding the default value takes away from the fact that in practice, the timeout can constantly change.

Reply:
Done.